src/libraries/Common/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -19,10 +21,10 @@ internal static partial class ADP
     {
         // NOTE: Initializing a Task in SQL CLR requires the "UNSAFE" permission set (https://docs.microsoft.com/en-us/dotnet/framework/performance/sql-server-programming-and-host-protection-attributes)
```

Comment:
netcore cannot and will never be hostable in sql server. the major requirement for hosting it is the runtime relibility infrastructure that has been removed from core because of the vast complexity it introduced for such low benefit. It doesn't need to be in this PR but this could be simplified to just static readonly tasks without the manual late init.

Reply:
Yeah, there's lots we can clean up in S.D.C., but am trying to keep this PR and discussion focused on nullability only. I do plan a pass after this, of course you're also welcome to submit a PR.